include/SugarFolders/SugarFolders.php

```diff
@@ -682,6 +687,34 @@ public function getFoldersForSettings($focusUser = null)
         return $ret;
     }
 
+    /**
+     * Remove folders of deleted inbounds
+     *
+     * @param array $folders - array of folders table rows
+     * @return array
+     */
+    private function removeDeletedFolders($folders) {
+
+        $ret = array();
+
+        foreach($folders as $folder) {
+            $correct = false;
+            if(!$folder['id']) {
```

Comment:
> Why are we using $correct twice? what is correct? The id?

No, correct each one which are not deleted. See the original function. Only different is: I need the non-deleted folders, so each one is correct if no folder[id] and the IE account exists.